src/invoices/AdjustmentsForm/AdjustmentsForm.js

```diff
@@ -126,6 +126,12 @@ const AdjustmentsForm = ({
       change(`${elem}.prorate`, value);
     };
 
+    const onExportToAccountingChange = ({ target: { checked } }) => {
+      change(`${elem}.exportToAccounting`, checked);
+
+      if (checked) change('exportToAccounting', checked);
```

Comment:
as I remember Dennis mentioned not to do this

Reply:
Maybe I missed it. thanks